scripts/measures/enrich-measures-data.js

```diff
@@ -100,7 +100,7 @@ function enrichStratifications(measures) {
     .filter(measure => measure.category === QUALITY_CATEGORY)
     .forEach(measure => {
       const stratification = stratifications.find(stratum => stratum.eMeasureId === measure.eMeasureId);
-      if (stratification) {
+      if (stratification && stratification.mapping) {
```

Comment:
@saquino0827 can you take a quick look at this-- I got an error when running this script and this seems like the right fix; some of the strata are being removed from measures-data.json but that looks like a consequence of two of these measures switching from nonproportional to singleperformancerate? Anyway I figure you're more familiar with the intent. Thanks!